clang/docs/analyzer/checkers.rst

```diff
@@ -2097,6 +2097,21 @@ This checker is a part of ``core.StackAddressEscape``, but is temporarily disabl
 
 .. _alpha-core-TestAfterDivZero:
 
+alpha.core.StdVariant (C++)
+"""""""""""""""""""""""""""
+Check if a value of active type is retrieved from an ``std::variant`` instance with ``std::get``.
+In case of bad variant type access(the accessed type differs from the active type)
```

Comment:
```suggestion
In case of bad variant type access (the accessed type differs from the active type)
```

Reply:
This diff is still undone, or was reverted / reappeared during the intermittent changes.